sky/users/server.py

```diff
@@ -34,7 +34,7 @@
 
 
 @router.get('')
-async def users() -> List[Dict[str, Any]]:
+def users() -> List[Dict[str, Any]]:
```

Comment:
Instead of moving it out of event loop (not sure how it affects the uvicorn), should just call schedule request just like our get_clusters?